### ISSUE TITLE
Missing Icon

### ISSUE THREAD

**Issue by fwon2638** [maxrave-dev/SimpMusic#278]
Sometimes when AA starts, SimpMusic icon does not appear. I have to disconnect and reconnect AA before SimpMucis cion appears on AA.

**Comment by maxrave-dev** [comment]
Other apps work?

**Comment by fwon2638** [comment]
Yes. From time to time, your app simply disappears from the AA launcher on startup. I have to disconnect and reconnect to make it appear again.
